e2e-polybft/framework/test-cluster.go

```diff
@@ -764,19 +767,23 @@ func NewTestCluster(t *testing.T, validatorsCount int, opts ...ClusterOption) *T
 		require.NoError(t, err)
 
 		// fund addresses on the rootchain
-		err = cluster.Bridge.fundAddressesOnRoot(polybftConfig)
+		err = bridge.fundAddressesOnRoot(polybftConfig)
 		require.NoError(t, err)
 
 		// add premine if token is non-mintable
-		err = cluster.Bridge.mintNativeRootToken(addresses, tokenConfig, polybftConfig)
+		err = bridge.mintNativeRootToken(addresses, tokenConfig, polybftConfig)
```

Comment:
We will only need to mint and premine native tokens, and execute `finalizeGenesis` on one external chain. We can not have multiple native tokens originating from multiple chains. Only from one. Change this so that this is only done on the first configured bridge.